test/cluster/failover.js

```diff
@@ -3,45 +3,45 @@
 const InstanceManager = require("../../InstanceManager.js");
 const expect = require("chai").expect;
 const arangojs = require("arangojs");
-const {sleep, afterEachCleanup} = require('../../utils');
+const { sleep, afterEachCleanup } = require("../../utils");
 
 // Arango error code for "shutdown in progress"
 const ERROR_SHUTTING_DOWN = 30;
+const CLUSTER_BACKEND_UNAVAILABLE = 1478;
 
-
-describe.skip("Failover", function() {
+describe("Failover", function() {
   const instanceManager = InstanceManager.create();
   let db;
 
   const getLeader = async function() {
-    const data = await InstanceManager
-      .rpAgency({
-        method: "POST",
-        url:
-          instanceManager.getEndpointUrl(instanceManager.agents()[0]) +
-          "/_api/agency/read",
-        json: true,
-        body: [["/"]]
-      });
+    const data = await InstanceManager.rpAgency({
+      method: "POST",
+      url:
+        instanceManager.getEndpointUrl(instanceManager.agents()[0]) +
+        "/_api/agency/read",
+      json: true,
+      body: [["/"]],
+    });
```

Comment:
Let's create a formatting scheme for our JS code and apply it to every supported version